content/en/docs/tutorials/kubernetes-basics/_index.html

```diff
@@ -24,7 +24,7 @@
     <div class="row">
       <div class="col-md-9">
         <h2>Kubernetes Basics</h2>
-        <p>This tutorial provides a walkthrough of the basics of the Kubernetes cluster orchestration system. Each module contains some background information on major Kubernetes features and concepts, and includes an interactive online tutorial. These interactive tutorials let you manage a simple cluster and its containerized applications for yourself.</p>
+        <p>This tutorial provides a walkthrough of Kubernetes (or K8s) cluster orchestration system basics. Each module contains some background information on major Kubernetes features and concepts, and includes an interactive online tutorial. These interactive tutorials let you manage a simple cluster and its containerized applications for yourself.</p>
```

Comment:
To clarify my earlier comments: The original grammar here, `This tutorial provides a walkthrough of the basics of the Kubernetes cluster orchestration system.` is more correct than this.